### ISSUE TITLE
Add hablity to change specific axis of actor position when select many actors

### ISSUE THREAD

**Issue by RuanLucasGD** [FlaxEngine/FlaxEngine#1056]
I can't change a specific axis of the actor when i have many actors selected. If i try to do this all the others axes of all actors are reseted.

**Unigine:**
[Desktop 2023.04.29 - 13.40.36.08.webm](https://user-images.githubusercontent.com/79365912/235317066-861586b0-625e-4641-b23f-bdb587be83ec.webm)

**Flax:**
[Debug 2023.04.29 - 13.41.31.09.webm](https://user-images.githubusercontent.com/79365912/235317119-fd4242bd-edf7-408d-8c94-fabcc0fc6d88.webm)

**Flax version:**
1.6

**Comment by mafiesto4** [comment]
Yeah, that's missing functionality which would be useful to have.

Source code location that handled `Vector3` editing with multiple values: https://github.com/FlaxEngine/FlaxEngine/blob/master/Source/Editor/CustomEditors/Editors/Vector3Editor.cs#L174

**Comment by Tryibion** [comment]
Added in #2870
